models/issue_comment.go

```diff
@@ -420,6 +422,27 @@ func (c *Comment) LoadAssigneeUser() error {
 	return nil
 }
 
+// LoadResolveDoer if comment.Type is CommentTypeCode and ResolveDoerID not zero, then load resolveDoer
+func (c *Comment) LoadResolveDoer() (err error) {
+	if c.ResolveDoerID == 0 || c.Type != CommentTypeCode {
+		return nil
+	}
+	c.ResolveDoer, err = getUserByID(x, c.ResolveDoerID)
+	if err != nil {
+		if IsErrUserNotExist(err) {
+			c.ResolveDoerID = -1
```

Comment:
I'm not sure resetting `ResolveDoerID` to `-1` is required. Do you have a reason to do so?

Reply:
yes, It's really not necessary.